pako/pako_manager.py

```diff
@@ -88,18 +100,18 @@ def install_one(self, package: str, fmt: str = None) -> bool:
                         possible_names.append(f)
 
         for name in possible_names:
-            if self.call(self.config['install'].split() + [name.format(package)]) == 0:
+            if self.call(self.config[action].split() + [name.format(package)]) == 0:
                 return True
         return False
+        
+    def install_one(self, package: str, fmt: str = None) -> bool:
+        PakoManager.action_one(self,"install",package,fmt=None)
```

Comment:
Couple of things with this one:
- Should these pass on any return values? 
- Any reason not to use `self` instead of `PakoManager`?
- Also, I don't think we want the `fmt=None` here. If no `fmt` argument is passed to `install_one()` then it will get the default `None` value anyway.

So I'd see it being:
```Python
return self.action_one(self, "install", package, fmt)
```